doc/TechnicalOverview.md

```diff
@@ -77,8 +77,8 @@ a fully qualified URL.
 Monsteer extends the spike reader and writer interfaces defined in the Brion plugin
 API. When a library is linked against the Monsteer library, those plugins are
 automatically instantiated. Here Monsteer library provides schema starting with
-"Monsteer" keyword, such as "monsteer://" for spike streaming and "MONSTEER_steer://" for
-steering.
+"Monsteer" keyword, such as "monsteer://" for spike streaming and
+"nestSimulator://" for steering.
```

Comment:
yes, it appears that the simulator is another type of lunchbox plugins + interface created with the idea of providing transparent support for different type of simulators (currently nest only)